### DISCUSSION TITLE
How to delete old recordings?

### DISCUSSION THREAD

**Discussion by CancunManny** [blakeblackshear/frigate#15496] [Ask A Question]
So everything was running smooth (had uptime of over 3 months) until xmas lights happened.  I tried masks, zones, etc., but I am guessing because of the white walls blinking lights keep triggering recordings.  Not that big of a deal until frigate stopped recording events.  The hard drive got full.

I read a recent post about frigate deleting all videos every 10 minutes or so, but on my end seems hard drive got full and it just stopped recording events.

So far I went in and changed motion recordings to only hold for 1 day (it was 5), and for events I changed from 14 to 3.  I rebooted the machine, but my hard drive was still full.

I then selected and deleted all events and detections, and rebooted, but the hard drive was still full.  I am running frigate on proxmox.  I had 83 gigs storage space, but bumped it up to 130 gigs.  It is recording again, but still shows over 60 gigs being used, even though it should be much less than that since I deleted all the events/detections.

![image](https://github.com/user-attachments/assets/ecd2c234-a027-4176-8a68-308159a8a45b)

The picture shows I am using 81 gigs of the 130.

However on the frigate storage are this is what I have

![image](https://github.com/user-attachments/assets/f34c9896-51aa-4186-96aa-66ae5a3e2596)

If I am reading it correctly it says I am only using about 3.25 gigs

Any ideas on how I can clean up the files on the hard drive?

**Comment by NickM-27** [comment]
> Any ideas on how I can clean up the files on the hard drive?

you just need to delete them manually, seems they didn't delete for some reason

**Comment by CancunManny** [comment]
Seems I will be better of saving my config files, and re installing frigate?

![image](https://github.com/user-attachments/assets/ff0963dd-344d-40ae-aa19-b36b17057392)

The storage folder seems to be 77 gigs, but the recordings folder is the one that supposed to have the actual recordings so it should be the largest folder, but when I check out recordings folder it only has a folder for today ( which would make sense since I deleted everything yesterday).  Seems the frigate.db is the largest file.

Is there something I am not seeing/noticing?  Should I just do a fresh install of Frigate instead?

**Reply by NickM-27** [reply]
that's a very odd size for the DB, you should just delete the DB file in that case

**Reply by CancunManny** [reply]
if I delete de DB will frigate recreate it itself, or would I have to install everything from scratch?

**Reply by NickM-27** [reply]
Yes frigate will recreate it
